runners/spark/src/main/java/org/apache/beam/runners/spark/translation/EvaluationContext.java

```diff
@@ -224,20 +223,15 @@ public MetricResults metrics() {
     return boundedDataset.getValues(pcollection);
   }
 
-  @Override
-  public void close(boolean gracefully) {
-    if (isStreamingPipeline()) {
-      // stop streaming context
-      if (timeout > 0) {
-        jssc.awaitTerminationOrTimeout(timeout);
-      } else {
-        jssc.awaitTermination();
+  @Override public void close(boolean gracefully) {
```

Comment:
While it's a matter of convenience, SparkRunner code usually puts `@Override` above the method.